pybluetooth/pyusb_bt_sockets.py

```diff
@@ -60,6 +60,12 @@ def __del__(self):
             self.hci_reset()
         except:
             pass
+        # Release the device, so it can be claimed again immediately when
+        # this object gets free'd.
+        try:
+            usb.util.dispose_resources(self.pyusb_dev)
+        except:
+            pass
```

Comment:
Maybe log here? The `except: pass` pattern has burned me a few times in the past.